typescript-sdk/src/observability/setup/config.ts

```diff
@@ -0,0 +1,38 @@
+import { Logger, NoOpLogger } from "../../logger/index.js";
+import { ObservabilityConfig } from "./types";
+
+let observabilityConfig: ObservabilityConfig | null = null;
+
+export function setObservabilityConfig(config: ObservabilityConfig) {
+  if (observabilityConfig) {
+    observabilityConfig.logger.error("[LangWatch Observability] Observability config already set; skipping");
+    return;
+  }
+  observabilityConfig = config;
+}
+
+export function setObservabilityConfigInstance(instance: ObservabilityConfig | null) {
+  observabilityConfig = instance;
+}
+
+export function getObservabilityConfig(): ObservabilityConfig {
+  if (!observabilityConfig) {
+    console.error("[LangWatch Observability] Please call setupObservability() before using the LangWatch Observability API");
+    return {
+      logger: new NoOpLogger(),
+    };
+  }
+  return observabilityConfig;
+}
+
+export function getObservabilityLogger(): Logger {
+  return getObservabilityConfig().logger;
+}
+
+export function getObservabilityConfigSuppressInputCapture(): boolean {
+  return getObservabilityConfig().suppressInputCapture ?? false;
+}
+
+export function getObservabilityConfigSuppressOutputCapture(): boolean {
+  return getObservabilityConfig().suppressOutputCapture ?? false;
+}
```

Comment:
Consider docstrings/explainations for these exported methods

Reply:
Also, when I see a group of functions with all the same name calling each other, it seems like a class to me.  It's also managing state with a module variable in memory, which I think is the base argument for a class (holding state): 
```
let observabilityConfig: ObservabilityConfig | null = null;
```